Sources/Preferences/PreferencesTabViewController.swift

```diff
@@ -93,6 +94,26 @@ final class PreferencesTabViewController: NSViewController, PreferencesStyleCont
 		}
 	}
 
+	private func updateWindowTitle(tabIndex: Int) {
+		self.window.title = {
+			if preferencePanes.count > 1 {
+				return preferencePanes[tabIndex].preferencePaneTitle
+			} else {
+				let appBundle = Bundle.main
+				let appName = appBundle.localizedInfoDictionary?["CFBundleDisplayName"] as? String
+					?? appBundle.infoDictionary?["CFBundleDisplayName"] as? String
+					?? appBundle.localizedInfoDictionary?["CFBundleExecutable"] as? String
+					?? appBundle.infoDictionary?["CFBundleExecutable"] as? String
+				let preferences = String(System.localizedString(forKey: "Preferences…").dropLast())
```

Comment:
I think we should explicitly document this behavior in the readme and link to the HIG. That when there are more than one pane, it uses the pane title, otherwise `App Name Preferences`.